Cargo.toml

```diff
@@ -22,13 +22,12 @@ default = []
 all = []
 
 [dependencies]
-hyper = "0.13"
+hyper = { version = "0.14", default-features = false, features = ["full"] }
```

Comment:
@rousan Do you know which features of hyper that Routerify is using? I specified full as a catch-all, but maybe we can pare it down.

Reply:
We can keep it as it is for now.